msamp/deepspeed/runtime/config.py

```diff
@@ -12,6 +12,8 @@
 MSAMP_ENABLED_DEFAULT = False
 MSAMP_OPTLEVEL = 'opt_level'
 MSAMP_OPTLEVEL_DEFAULT = 'O1'
+MSAMP_USETE = 'use_te'
```

Comment:
Please rename it with `MSAMP_USE_TE` and `MSAMP_USE_TE_DEFAULT`.

Reply:
Other choices: `MSAMP_TE_ENABLED`

Reply:
Replaced `MSAMP_USETE` and `MSAMP_USETE_DEFAULT` with `MSAMP_USE_TE` and `MSAMP_USE_TE_DEFAULT` respectively.